scripts/utils/InsertNuGetIntoSdkAndCli.ps1

```diff
@@ -13,8 +13,8 @@ PersonalAccessToken of the NuGetLurker account
 .PARAMETER RepositoryName
 The Repository to insert into (SDK or CLI)
 
-.PARAMETER BranchName
-The repository's branch to insert into
+.PARAMETER BranchNames
+Semicolon separated list of the repository's branches to insert into
```

Comment:
You probably need to fix what the release definition calls. 
I would not change the name, as you would break the 4.9.x releases.